airbyte-webapp/src/utils/cookiesUtils.ts

```diff
@@ -0,0 +1,13 @@
+import Cookies from "js-cookie";
+
+export const setSegmentAnonymousIdCookie = (queryString?: string): void => {
+  const queryParams = new URLSearchParams(queryString);
+  const ajs_anonymous_id = queryParams.get("ajs_anonymous_id");
+  if (ajs_anonymous_id) {
+    Cookies.set("ajs_anonymous_id", ajs_anonymous_id);
```

Comment:
I'd suggest we save this in local storage instead, so we wouldn't need to add a new library here for cookies.

Reply:
I went with the option of storing it directly in cookies because I think it will be better to override Segment cookie instead of having two separate instances of `ajs_anonymous_id`

Reply:
I personally feel at that point we'd be "reusing" and overwriting something that is internal to how Segment works, and I feel just using our own local storage flag and just communicate with Segment via their API (which we anyway still do), feels like the cleaner approach of not running into any issues (e.g. what if the segment library actually decides overwriting the `ajs_anonymous_id` cookie again in some scenarios with their own value, and we lost our original one).

That said, given that we're planning to remove this code in a couple of weeks again once we moved to cloud.airbyte.com I'm fine leaving this however you prefer :)

Reply:
Need to withdraw my comment :D I tested this, it actually is exactly what happens for me, the `ajs_anonymous_id` cookie is set from the query but then shortly afterwards rewritten (by segment) to a random UUID again, and thus we'd lose the parameter. So I think we'll need to store it in localsStorage.

Reply:
https://user-images.githubusercontent.com/877229/191519158-767a4e12-5ff6-4e8b-b4a1-c268a44dba1d.mp4

Reply:
I will double check this, because I'm not seeing Segment overriding it:
https://www.loom.com/share/e305ec6d955b49fe9e87cd6d21008b89

Trying your localStorage proposal was not working as Segment was indeed creating a new `ajs_anonymous_id`

I'll keep working on this